tests/Fixtures/Entity/SluggableWithoutRegenerateEntity.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Knp\DoctrineBehaviors\Tests\Fixtures\Entity;
+
+use DateTime;
+use DateTimeInterface;
+use Doctrine\ORM\Mapping as ORM;
+use Knp\DoctrineBehaviors\Contract\Entity\SluggableInterface;
+use Knp\DoctrineBehaviors\Model\Sluggable\SluggableTrait;
+
+/**
+ * @ORM\Entity
+ */
+class SluggableWithoutRegenerateEntity implements SluggableInterface
+{
+    use SluggableTrait;
+
+    /**
+     * @ORM\Column(type="string")
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="AUTO")
+     * @var int
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="datetime")
+     * @var DateTimeInterface
+     */
+    private $date;
```

Comment:
Where is this property used?

Reply:
It is not but I keep it from the SluggableEntity I copied. The goal of this property is to have an other property which is not concerned by the slug generation. For the moment, this property is useless, so we have 3 choices:
- Keep it for future test case
- Remove it because there is no test based on it
- Add test concerning it like in Knp\DoctrineBehaviors\Tests\ORM\SluggableTest class

What do you think?

Reply:
I'd remove it in both, because it doesn't add any value

Reply:
The value of the createdAt property in Knp\DoctrineBehaviors\Tests\Fixtures\Entity\SluggableEntity is to test that the slug is not updated when we change an unrelated value.
But in Knp\DoctrineBehaviors\Tests\Fixtures\Entity\SluggableWithoutRegenerateEntity, there is no test with this createdAt property so I agree to delete it.